Makefile

```diff
@@ -305,7 +305,7 @@ endif
 
 # Get all clojure files, including untracked, excluding removed
 define find_all_clojure_files
-$$(comm -23 <(sort <(git ls-files --cached --others --exclude-standard)) <(sort <(git ls-files --deleted)) | grep -e \.clj$$ -e \.cljs$$ -e \.cljc$$ -e \.edn)
+$$(comm -23 <(sort <(git ls-files --cached --others --exclude-standard)) <(sort <(git ls-files --deleted)) | grep -e '^(?!clj-kondo).*\.clj'$$ -e \.cljs$$ -e \.cljc$$ -e '^(?!clj-kondo).*\.clj')
```

Comment:
I'm not 100% sold that this will work. I'll do a few tests and get back to you.

Reply:
Sorry I had a typo, Will push a fix.

Reply:
nice!